server/redis.repository.js

```diff
@@ -0,0 +1,79 @@
+const redis = require("redis");
+const { promisify } = require("util");
+const client = redis.createClient();
+const hgetAsync = promisify(client.hget).bind(client);
+const hincrAsync = promisify(client.hincrby).bind(client);
```

Comment:
should we rename this to hincrybyAsync for consistency?

Reply:
+1 from me for this point

Reply:
Good catch!